http/preview.go

```diff
@@ -35,9 +35,6 @@ type FileCache interface {
 
 func previewHandler(imgSvc ImgService, fileCache FileCache, enableThumbnails, resizePreview bool) handleFunc {
 	return withUser(func(w http.ResponseWriter, r *http.Request, d *data) (int, error) {
-		if !d.user.Perm.Download {
-			return http.StatusAccepted, nil
-		}
```

Comment:
This means that the user can download any image from the server without corresponding permission.

Reply:
Currently the image files are listed, but shows broken img elements on the listing and previewer. With this change, users that dont have the download permission will be allowed to view resized images (thumbnails and previews), but still wont be able to download the raw image. This follows the concept of text files, that users without download permission can view the content. The main idea would be allowing users to view any file handled by the previewer.

Do you think that change should be removed?

Reply:
If resizing is disabled, the previewer will return the original image. The same happens when the HD button is pressed. Giving users access to the content but not letting them download it doesn't make sense. Nothing stops a user from pressing the right button and save the image. In my opinion, previewing text files without download permission is a bug.

Reply:
I see, changing the behavior of unprivileged users can also be unexpected to servers admins, since it currently have no customization. I will left it as is since would require a overhaul to make it fit for all use causes.